source/docs/advanced/performance/preloading-routes.md

```diff
@@ -0,0 +1,183 @@
+---
+id: preloading-routes
+title: Preloading Routes' data
+---
+
+When a website is slow, the first thing that comes to mind is to improve raw metrics like the Speed Index. And while this is definitly useful, we shouldn't overlook the [_perceived_ performances](https://blog.teamtreehouse.com/perceived-performance). The goal is not to improve the response time, but to better manage the time to make the user _think_ that the website is fast.
+
+One way to do this is to anticipate the user's need and start the long running tasks before they actually need it. For instance, when a user is navigating on a catalog, on desktop, they will hover the product before clicking on it. Thus, we can trigger the product's load as soon as they hover the product, instead of waiting for the click. This can seem like small gains, but this definitly improves how fast the user _thinks_ your site is.
+
+## How to preload data?
+
+The main idea is to launch Apollo requests twice:
+
+- when you think that the user will need the data
+- when you are displaying the data
+
+By doing this, you will first warm up Apollo's cache and this cache will be reused when you will display your data. But this can be tedious. This is why in Front-Commerce, we have added a <abbr title="Higher Order Component">HOC</abbr> that will do this: `web/core/apollo/graphqlWithPreload` (added in version 1.0.0-rc.0).
```

Comment:
```suggestion
By doing this, you will first warm up Apollo's cache and this cache will be reused when you will display your data. But this can be tedious. This is why in Front-Commerce, we have added a <abbr title="Higher Order Component">HOC</abbr> that will do this: `web/core/apollo/graphqlWithPreload`.
```